hack/update-codegen.sh

```diff
@@ -17,8 +17,69 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
-CODEGEN_PKG="${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}"
-"${CODEGEN_PKG}"/generate-groups.sh "deepcopy,defaulter,client,lister,informer" \
-  k8s.io/test-infra/prow/client k8s.io/test-infra/prow/apis \
-  prowjobs:v1
+cd "$(git rev-parse --show-toplevel)"
+
+export GOPATH="${GOPATH:-$HOME/go}"
+export PATH="$GOPATH/bin:$PATH"
+export GO111MODULE=on
+
+ensure-in-gopath() {
+  if [[ "$PWD" != "$GOPATH/src/k8s.io/test-infra" ]]; then
+    echo Sadly, $(basename "$0") most run inside GOPATH=$GOPATH, not $PWD >&2
+    exit 1
+  fi
+}
+
+codegen-init() {
+  echo "Ensuring generators exist..." >&2
+  local ver=b1289fc74931d4b6b04bd1a259acfc88a2cb0a66
```

Comment:
where does this version come from? should it be more visible?

Reply:
The version we're using now. No